### DISCUSSION TITLE
What is the best way to draw the cleavage of a protein?

### DISCUSSION THREAD

**Discussion by danidi** [wikipathways/wikipathways-help#38] [Q&A] [answered]
Hi, 
I'm currently trying to figure out what the best way to draw the cleavage of a protein/peptide would be. My idea would be to use mim-conversion from the uncut to the cut protein, and a mim-cleavage from the enzyme to the conversion interaction. However, what node and identifier should I use for the cut protein? If I use a protein data node with the ID of the original protein and specify the sequence positions in the comment, would this cause issues when used for analysis (i.e. the protein is converted to itself)? I could also add it without identifier, or even draw it as a graphical element, but then I would lose information.
Do you have other ideas or suggestions? 
Thanks, Daniela

**Comment by Chris-Evelo** [comment, marked as answer]
That is a really interesting question. In some cases, the products of such a cleavage are actually known and have separate protein IDs (where BridgeDb might actually map to the same gene for all, but that is a separate problem). In other cases, the protein product(s) might be known and yet have no identifiers. Maybe then we should mint such IDs in WikiData? Finally, there are many cases where cleavage leads to different and unspecified products. In such a case we should only use a label like "peptides" I would think and not try to add an ID to something that is not unique.

**Reply by danidi** [reply]
Thank you! In this case, there are a few specific fragments, e.g. Tau 1–368. I couldn't find an identifier for it, but maybe I was searching in the wrong places.

**Comment by Chris-Evelo** [comment]
Are they in PeptideAtlas? https://peptideatlas.org/ If so, would that provide a solution?

**Reply by danidi** [reply]
Nice database, I wasn't aware of it. Unfortunately I couldn't find this specific case, they seem to have only smaller fragments. But it will be useful in other cases.

**Comment by khanspers** [comment]
I agree with Chris that if an identifier for the specific fragment is not known, it's better to leave it unspecified. 

In terms of how to represent it draw it, the option below is what we agreed upon in the Curator's meeting on 7/10:

![Screen Shot 2023-07-10 at 4 42 37 PM](https://github.com/wikipathways/wikipathways-help/assets/1442197/58d6d28e-a57b-458e-a63b-27184fe3eb0c)
